cluster/common.sh

```diff
@@ -1270,6 +1270,7 @@ function parse-master-env() {
   REQUESTHEADER_CA_CERT_BASE64=$(get-env-val "${master_env}" "REQUESTHEADER_CA_CERT")
   PROXY_CLIENT_CERT_BASE64=$(get-env-val "${master_env}" "PROXY_CLIENT_CERT")
   PROXY_CLIENT_KEY_BASE64=$(get-env-val "${master_env}" "PROXY_CLIENT_KEY")
+  ENABLE_LEGACY_ABAC=$(get-env-val "${master_env}" "ENABLE_LEGACY_ABAC")
```

Comment:
Does this set it to "" and prevent the defaulting in config-default?

Reply:
Yes, which means the only way to change this on an existing cluster is to modify the existing kube-env in metadata.

I'm not sure of a better way to handle updates. Either, we do this, and changing the existing value is kinda gross, or we go off of the config-default value, and we (possibly surprisingly) update the value on existing clusters. This felt like the less intrusive approach.

Reply:
Is there a well defined procedure for changing existing cluster parameters generally?

Reply:
we definitely should not update the value on existing clusters during upgrade. existing workloads should continue to run as-is on an upgraded cluster.

updating the default for new clusters is what I'd like to see, though the interaction between metadata env and the config-default.sh scripts is not clear to me (I can't tell which runs first, which informs the other, etc).